### DISCUSSION TITLE
感谢作者提供的这么好的工具

### DISCUSSION THREAD

**Discussion by ssyangchn** [XIU2/CloudflareSpeedTest#351] [讨论]
谢啦!!☆⌒(*＾-゜)v

**Comment by XIU2** [comment]
emmm 不用谢。

觉得好用的话，就点个⭐吧！

**Comment by ssyangchn** [comment]
> emmm 不用谢。
> 
> 觉得好用的话，就点个⭐吧！

正在看源码学习

**Comment by XIU2** [comment]
我源码写的很烂，毕竟是看了下基础教程就直接上手了，新手水平。

如果没有其他问题的话，我就先关闭该 Issues 了，有问题可以继续在这里回复。
